scripts/oneoff/generate-dos-2-master-csv.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/python
```

Comment:
On top of this, I don't really understand why we need `oneoff` scripts for these anymore if all that's changing between them is the framework slugs.